agent/containers/images/image-test-script.sh

```diff
@@ -4,65 +4,49 @@
 # Redis server, and one or more remote
 export CONTROLLER=$(hostname -f)
 
-verified=""
-while [ -z "$verified" ]
-do
-    read -p "Specify distribution to test (or press 'ENTER' for default 'centos-8'): " DISTRO
-    if [ -z "$DISTRO" ]
-    then
-        DISTRO="centos-8"
-    fi
+DISTRO_DEFAULT="centos-8"
+TAG_DEFAULT="a933ae45e"
+REDIS_HOST_DEFAULT=$CONTROLLER
+REDIS_PORT_DEFAULT=17001
 
-    read -p "Specify image tag to test (or press 'ENTER' for default 'a933ae45e'): " TAG
-    if [ -z "$TAG" ]
-    then
-        TAG="a933ae45e"
-    fi
+while :; do
+    read -p "Specify distribution to test (or press 'ENTER' for default ${DISTRO_DEFAULT}): "
+    DISTRO=${REPLY:-${DISTRO_DEFAULT}}
+    read -p "Specify image tag to test (or press 'ENTER' for default ${TAG_DEFAULT}): "
+    TAG=${REPLY:-${TAG_DEFAULT}}
 
     printf -- "Pulling tool-data-sink image to test...\n"
-    verified=$(podman pull quay.io/pbench/pbench-agent-tool-data-sink-$DISTRO:$TAG 2> /dev/null)
-
-    if [ -z "$verified" ]
+    if podman pull quay.io/pbench/pbench-agent-tool-data-sink-$DISTRO:$TAG
     then
-        printf -- "Distro/tag combination could not be found, please try again.\n\n"
+        printf -- "Complete!\n\n" break
+        break
     else
-        printf -- "Complete!\n\n"
+        printf -- "Distro/tag combination could not be found, please try again.\n\n"
     fi
 done
 
-badhost=true
-while [ "$badhost" == true ]
-do
-    read -p "Specify redis host (or press 'ENTER' for default '$CONTROLLER'): " REDIS_HOST
-    if [ -z "$REDIS_HOST" ]
+while :; do
+    read -p "Specify redis host (or press 'ENTER' for default '$REDIS_HOST_DEFAULT'): "
+    REDIS_HOST=${REPLY:-${REDIS_HOST_DEFAULT}}
+    if ping -c 1 -W 1 ${REDIS_HOST} 2>&1 1>/dev/null
     then
-        REDIS_HOST=$CONTROLLER
-    fi
-    ping_first=$(ping -c 1 -W 1 ${REDIS_HOST} 2>&1 | grep 'Name or service not known')
-    ping_second=$(ping -c 1 -W 1 ${REDIS_HOST} 2>&1 | grep '100% packet loss')
-    if [ -n "$ping_first" ] || [ -n "$ping_second" ]
-    then
-        printf -- "Host specified either does not exist or is unreachable, please try again:\n"
+        break
```

Comment:
Since we're validating successful tag/container validation above, should we also be explicitly confirming Redis host validation/reachability here? Should we announce up front that we're pinging the Redis host?